### ISSUE TITLE
LoadFromStream/SaveToStream encoding inconsistency

### ISSUE THREAD

**Issue by pyscripter** [EtheaDev/SVGIconImageList#18]
```pascal
TSVG.LoadFromStream:

    SL.LoadFromStream(Stream, TEncoding.UTF8);

TSVG.SaveToStream

    SL.SaveToStream(Stream);

```
SaveToStream will use the default encoding which is ANSI on Windows.   So sgv text is written as ansi and read as utf8.   
To make consistent save as utf8 as well.

**Comment by carloBarazzetta** [comment]
Right.

**Comment by carloBarazzetta** [comment]
Resolved into 1.7 version.
